src/Configuration/Dotenv/Internal.hs

```diff
@@ -0,0 +1,14 @@
+module Configuration.Dotenv.Internal
+  ( module Configuration.Dotenv.Types
```

Comment:
@CristhianMotoche looks good, but I'd recommend only exporting the functions and types needed for testing to hide some implementation details; the more implementation details we expose, the greater the risk of introducing a breaking change.

Reply:
Makes sense to me @sestrella I'll keep the export of `Configuration.Dotenv.Environment` to avoid the major release. Please, take a second look at it. Thanks in advance!